src/mockData/bb6a5e451dac.ts

```diff
@@ -832,7 +832,7 @@ export const comparisonResults: CompareResultsItem[] = [
   {
     base_rev: '3639b6891b332ea6c4ffea87bb2d1139c38b781d',
     new_rev: 'bb6a5e451dace3b9c7be42d24c9272738d73e6db',
-    framework_id: 1,
+    framework_id: 16,
```

Comment:
The changes to the mock can be removed if you use the fixtures instead ( `src/__tests__/utils/fixtures.ts` ).